package.json

```diff
@@ -23,18 +23,18 @@
   "author": "Tommy Chen <tommy351@gmail.com> (http://zespia.tw)",
   "license": "MIT",
   "dependencies": {
-    "nunjucks": "^2.4.1",
-    "object-assign": "^4.0.1"
+    "nunjucks": "^3.0.0",
+    "object-assign": "^4.1.1"
   },
   "devDependencies": {
     "chai": "^3.5.0",
-    "cheerio": "^0.20.0",
-    "eslint": "^2.4.0",
-    "eslint-config-hexo": "^1.0.3",
-    "hexo": "^3.0.0",
-    "jscs": "^2.11.0",
+    "cheerio": "^0.22.0",
+    "eslint": "^3.19.0",
```

Comment:
I suggest keeping these package in their old version. I assume the CI errors are caused by the new versions.